tests/app/celery/test_tasks.py

```diff
@@ -950,26 +950,48 @@ def test_save_sms_does_not_send_duplicate_and_does_not_put_in_retry_queue(sample
     assert not retry.called
 
 
-@pytest.mark.parametrize('personalisation', (
-    {
+@pytest.mark.parametrize('personalisation, expected_to, expected_normalised', (
+    ({
         'addressline1': 'Foo',
         'addressline2': 'Bar',
         'addressline3': 'Baz',
         'addressline4': 'Wibble',
         'addressline5': 'Wobble',
         'addressline6': 'Wubble',
         'postcode': 'SE1 2SA',
-    },
-    {
+    }, (
+        'Foo\n'
+        'Bar\n'
+        'Baz\n'
+        'Wibble\n'
+        'Wobble\n'
+        'Wubble\n'
+        'SE1 2SA'
+    ), (
+        'foobarbazwibblewobblewubblese12sa'
+    )),
+    ({
         # The address isn’t normalised when we store it in the
-        # `personalisation` column, but the first line is normalised for
-        # Storing in the `to` column
+        # `personalisation` column, but is normalised for storing in the
+        # `to` column
```

Comment:
This comment confuses me?
I thought we store it as is in the personalisation column, as a single string in the `to` column and as a normalised string in the `normalised_to` column? But this says we are storing it normalised in the `to` column?

Reply:
So I think what I've figured out is that there is actually two times we normalise the data
Once from 7 different strings to a single string that has spaces
A secondly from a single string that has spaces to a single string without spaces and lowercase
This then means that when we refer to `normalise` around the codebase to do it with, it's quite confusing and maybe a little misleading. Is there a quick fix to this? If it's a long one we may have to live with it though

Reply:
The two meanings are normalised for search and normalised for the postal system. Might have to live with it since it goes as deep as the name of a database column though 😬